package-res/resources/web/pentaho/visual/role/_doc/spec/IMapping.jsdoc

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright 2010 - 2016 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The `spec.IMapping` interface describes the _object_ form of a
+ * [visual role mapping]{@link pentaho.visual.role.Mapping} instance.
+ *
+ * @example
+ * {
+ *   level: "ordinal",
+ *   attributes: ["family", "product"]
+ * }
+ *
+ * @name pentaho.visual.role.spec.IMapping
+ *
+ * @interface
+ * @extends pentaho.type.spec.IComplex
+ *
+ * @see pentaho.visual.role.spec.IMappingAttribute
+ */
+
+/**
+ * The fixed measurement level in which the associated visual role is to operate on.
```

Comment:
Suggest:

The fixed measurement level on which the associated visual role is to operate.